Docker Scaffolding (#3)
JRaviLab/molevolvr2.0: README.md

# MolEvolvR Stack

This repo contains the implementation of the MolEvolvR stack, i.e.:

> - `app`: the frontend, written in Vue

Reviewer: ```suggestion
- `app`: the frontend webapp, written in React
```

- `backend`: a backend written in [Plumber](https://www.rplumber.io/index.html)
- `cluster`: the containerized SLURM "cluster" on which jobs are run
- `services`: a collection of services on which the stack relies:
    - `postgres`: configuration for a PostgreSQL database, which stores job information

Most of the data processing is accomplished via the `MolEvolvR` package, which
is currently available at https://github.com/JRaviLab/molevol_scripts. The stack
simply provides a user-friendly interface for accepting and monitoring the
progress of jobs, and orchestrates running the jobs on SLURM. The jobs
themselves call methods of the package at each stage of processing.

## Running the Stack in Development

> To run the stack, you will need to have Docker and Docker Compose installed.

Reviewer: ```suggestion
To run the stack, you'll need to [install Docker and Docker Compose](https://www.docker.com/).
```
